### DISCUSSION TITLE
Change or remove FUNDING.yml?

### DISCUSSION THREAD

**Discussion by BitesizedLion** [faker-js/faker#56] [General]
Currently the FUNDING.yml points to Marak (previous author whom is now banned from github).

Maybe this should be updated or removed?

**Comment by Shinigami92** [comment]
https://github.com/faker-js/faker.js/pull/30#issuecomment-1008278307

**Comment by damienwebdev** [comment]
https://github.com/faker-js/faker.js/pull/15#issuecomment-1008174977

**Comment by damienwebdev** [comment]
I'm going to post my thoughts here based upon a discussion that we had when initiating this repository:


> I'm strongly of the opinion that Marak's contributions are valuable and until such time that we actually need funding to maintain this repo, we leave his name in there.
> 
> I'm consciously leaving his name in there for the following three reasons:
> 
> 1. Marak's apartment burned down, but I'm not knowledgeable enough on the topic to say anything more. That said, there's no reason to kick someone while they're down.
> 
> 2. Marak has strong opinions on others using his work for a profit that I want to respect.
> 
> 3. Opinion/Speculation: Marak seems to be going through a tough time and I can't imagine taking away funding will make it any easier.

Hopefully, this is the last time that we as maintainers have to speak on the matter.

**Comment by JessicaSachs** [comment]
## Open Collective 
After going through some advisement by the Open Collective, as maintainers, we've made the decision to move the previously awarded funding of $11,652.69 into a [fakerjs-legacy](https://opencollective.com/fakerjs-legacy) collective that we do not have access to. This does not include any GitHub Sponsorships that are tied to Marak's personal Github account.

####  Transaction ledger on the fakerjs-legacy account on Open Collective [here](https://opencollective.com/fakerjs-legacy)

<img width="860" alt="Screen Shot 2022-01-12 at 1 52 28 PM" src="https://user-images.githubusercontent.com/2801156/149211111-b40bd5f8-306b-4822-a8dd-e1a6ba180f84.png">

## Moving forward
Now, going forward, our goal is to support this project and its maintenance in the best way possible. I would like to have a completely clean break as that's what I believe is healthiest for the project.

With regards to the Github Sponsorship button, in my opinion, we should route all funds to the same place: Open Collective. **However, I think that removing the Sponsorship button altogether is the first priority.** It is actually disingenuous to leave it in.

People will come to this project with all sorts of notions of what has or has not happened. Some people will have never seen this project before. I'm optimizing for the following scenarios...

**New User**

1. You use faker
2. You LOVE faker and want to sponsor its continued development.
3. You click on the pink heart and you pay money
4. You believe that you are sponsoring the continued maintenance of the project.

The issue is that you're not sponsoring the continued maintenance of the project. This conversation **could** be a moral debate, but it's really much simpler if we focus on solving the immediate problem.

Now, let's think bigger.

**Longtime User needs Bugfix Prioritized**

1. Your company uses faker
2. You have a bug in faker that you need prioritized
3. You remember hearing somewhere that the project really needs money
4. You convince your company to help you pay the maintainer to prioritize the bug.
5. Your company uses the GH Sponsors button to give 500$ to the maintainer in order to expedite the bugfix.
6. The bug fix does not happen and the money the company put toward the fix is gone.

There are so many problems that arise when we don't do the most sensible thing. We've already done a lot by voluntarily putting the $ into a completely separate account that we don't have access to.

We must be ingenuous and focus on the health of the project without compromising our user's trust (any further!). For folks on the outside looking in on this discussion, more clarity will come when I finish writing up the statement.

## Sponsorship of this project going forward
I will elaborate on this in the statement, but we're working with Open Collective via this workshop called "[How to Spend Your Money](https://opencollective.com/workshops/events/deciding-on-how-to-use-your-money-020765bf)" and will likely be modeling anything we do after ESLint's "[A year of paying contributors: Review](https://eslint.org/blog/2020/10/year-paying-contributors-review)"

**Reply by prisis** [reply]
Im in the same boat

**Reply by Shinigami92** [reply]
#15 #43 #110 #111 #112

Every second hour I need to mark an issue as duplicate...
Could we please just do something now... it gets annoying...

This, I assume, is exactly the effect @JessicaSachs is talking about...

**Reply by mmahalwy** [reply]
Agreed and supportive!
